.github/workflows/docker-test.yml

```diff
@@ -1,11 +1,15 @@
 name: Test Build-Env Docker Image
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
 
 on:
   pull_request:
     branches: master
     paths:
-    - '**Dockerfile'
-    - '**docker.yml'
+    - 'docker/**'
```

Comment:
I figured we want the docker tests to run if any file except for the README is changed, because the tests works in conjunction with the scripts and docker-compose.yml... Hmmm... but maybe different filters are needed for docker.yml.

Reply:
>but maybe different filters are needed for docker.yml.

I reverted the change below in docker.yml. In theory, if it passes this test, there's no reason to trigger it for the other one. The build should have already been triggered by any changes here.. or not...

I dunno. Let me know what you think. Or we could avoid some thought and adjust later as needed.

Reply:
I opened an issue so that I don't forget about this and we can delay the decision as I don't have strong feelings about one or the other option :D.